transformer_engine/pytorch/constants.py

```diff
@@ -14,7 +14,7 @@
 with enum in transformer_engine.h
 """
 TE_DType = {
-    torch.int8: tex.DType.kByte,
+    torch.uint8: tex.DType.kByte,
```

Comment:
I knew this was forceful - I replaced int8 with uint8 in this dict, instead of keeping them both for backward compatibility. But I do think anyone who develops FP8 code should stick to uint8 and discard any use of int8. I did a sweep for int8 at the time and switched them to uint8, but maybe this occurance in module.py came later?

Reply:
I changed all the other `torch.int8` to `torch.uint8`. Luckily they only occured in some of the Python files, no PyTorch-C type conversion occured.